.github/workflows/generate_new_client_hermetic_build.yaml

```diff
@@ -0,0 +1,157 @@
+name: Generate new GAPIC client library (Hermetic Build)
+on:
+  workflow_dispatch:
+      # some inputs are ommited due to limit of 10 input arguments
```

Comment:
I know this is an existing limitation of Github actions, but IMO this is a indication that Github action may not be a good fit for new client library generation. Because we are clearly omitting some parameters here, they may not be "required" to generate a new library, but `repo_metadata.json` would be missing some info without those parameters. 
I think we should find a different way sooner than later, but it would be out of scope for this PR. cc: @suztomo

Reply:
> repo_metadata.json would be missing some info without those parameters.

What are missing?

Reply:
For example, [these four](https://github.com/googleapis/google-cloud-java/blob/main/java-asset/.repo-metadata.json#L14-L17) in repo_metadata.json, we might be able to get them heuristically, but if we want to override them, there is no way other than manually adding them.

Reply:
As we discussed today, the long term solution is to 
1. Make the PR generation logic more generic so that it can recognize any changes to the generation config
2. Trigger the generation process on every generation config change.
3. Create a PR(Manually or still use this Github action) that adds a new section for the new client, which should trigger the process above.
CC: @JoeWang1127

Reply:
Thank you for the example.

> manually adding them.

The best way is to automatically get the data from the service config yaml.